packages/turf-center-median/index.js

```diff
@@ -0,0 +1,122 @@
+import centerMean from '@turf/center-mean';
+import distance from '@turf/distance';
+import centroid from '@turf/centroid';
+import { isNumber, point, isObject, featureCollection } from '@turf/helpers';
+import { featureEach } from '@turf/meta';
+import { getCoord } from '@turf/invariant';
+
+/**
+ * Takes a {@link FeatureCollection} of points and calculates the median center,
+ * algorithimically. It uses the algorithm described in:
+ *
+ * Harold W. Kuhn and Robert E. Kuenne, “An Efficient Algorithm for the
+ * Numerical Solution of the Generalized Weber Problem in Spatial
+ * Economics,” _Journal of Regional Science_ 4, no. 2 (1962): 21–33,
+ * doi:{@link https://doi.org/10.1111/j.1467-9787.1962.tb00902.x}.
+ *
+ * James E. Burt, Gerald M. Barber, and David L. Rigby, _Elementary
+ * Statistics for Geographers_, 3rd ed., New York: The Guilford
+ * Press, 2009, 150–151.
+ *
+ * Turfjs has four different functions for calculating the center of a set of
+ * data. Each is useful depending on circumstance.
+ *
+ * `@turf/center` finds the simple center of a dataset, by finding the
+ * midpoint between the extents of the data. That is, it divides in half the
+ * farthest east and farthest west point as well as the farthest north and
+ * farthest south.
+ *
+ * `@turf/center-of-mass` imagines that the dataset is a sheet of paper.
+ * The center of mass is where the sheet would balance on a fingertip.
+ *
+ * `@turf/center-mean` takes the averages of all the coordinates and
+ * produces a value that respects that. Unlike `@turf/center`, it is
+ * sensitive to clusters and outliers. It lands in the statistical middle of a
+ * dataset, not the geographical. It can also be weighted, meaning certain
+ * points are more important than others.
+ *
+ * `@turf/center-median` takes the mean center and tries to find, iteratively,
+ * a new point that requires the least amount of travel from all the points in
+ * the dataset. It is not as sensitive to outliers as `@turf/center`, but it is
+ * attracted to clustered data. It, too, can be weighted.
+ *
+ * @name centerMedian
+ * @param {FeatureCollection<Any>} features GeoJSON Feature or Geometry
```

Comment:
`FeatureCollection<Any>` means Polygon & LineString would also be accepted.. Do we just want to stick with Points for this one, the Typescript definition says otherwise so maybe this is a typo.

Reply:
No, it works for any. This is intentional 😂 . I think you see this by the end of the comments, though?

Reply:
👍 we should then apply the same description to the Typescript definition since that is the only file that will actually enforce the Input Types, the JSDocs is simply for the documentation and not used for enforcing types.

Reply:
I'll change it back to `any`